cg/src/utils/blockchain.js

```diff
@@ -176,6 +196,25 @@ async function listenerForRectificationEvent(callback) {
   });
 }
 
+async function listenerForRestrictionEvent(callback) {
+  const quorumContract = await getContract();
+  return await quorumContract.contract.events.Controller_SubjectDataRestricted((error, data) => {
+    if (error) {
+      winston.error(`Error handling restriction ${error.toString()}`);
+      return;
+    }
+
+    // There's a bug when a smart contract event is fired with a boolean parameter set to 'false', which transforms that boolean in 'null'.
+    // This piece of code corrects the nulled booleans, if there is any.
+    // I'm not sure why this bug happens, but it seems to be an issue with event firing in Ethereum/Quorum
+    const directMarketing = data.returnValues.directMarketing === null ? false : true;
```

Comment:
`data.returnValues.directMarketing === null ? false : true` => `data.returnValues.directMarketing !== null` ?

Reply:
Trying it out

Reply:
This is a good short-hand, thx